artwork/logo/coala_color.svg

```diff
@@ -331,15 +331,15 @@
      inkscape:pageopacity="0.0"
      inkscape:pageshadow="2"
      inkscape:zoom="11.313709"
-     inkscape:cx="7.3668075"
+     inkscape:cx="-13.537036"
```

Comment:
Upon upgrading the image using my inkscape instance, this happened

```diff

@@ -14,7 +14,7 @@
    height="48px"
    id="svg2985"
    version="1.1"
-   inkscape:version="0.91 r13725"
+   inkscape:version="0.92.3 (2405546, 2018-03-11)"
    sodipodi:docname="coala_logo.svg"
    inkscape:export-filename="E:\Bilder\Coala\coala.png"
    inkscape:export-xdpi="1124.9296"
@@ -339,9 +339,9 @@
      inkscape:document-units="px"
      inkscape:snap-global="false"
      inkscape:window-width="1366"
-     inkscape:window-height="709"
+     inkscape:window-height="744"
      inkscape:window-x="0"
-     inkscape:window-y="0"
+     inkscape:window-y="24"
      inkscape:window-maximized="1"
      inkscape:snap-bbox="true"
      inkscape:bbox-nodes="true"
```

and judging by the namespace and http://wiki.inkscape.org/wiki/index.php/Inkscape-specific_XML_attributes this change might not impact the end image at all.

Reply:
And also the editor does not complain if just the version number is changed.

Reply:
It would be nice if they documented these things a bit better.

Based on https://bugs.launchpad.net/inkscape/+bug/286745 , I think it is safe to remove this change also.